bim2sim/task/common/common.py

```diff
@@ -918,6 +931,32 @@ def _check_unique(inst, id_list):
             return True
         return id_list.count(inst.GlobalId) == 1
 
+    @staticmethod
+    def check_critical_uniqueness(id_list):
+        """
+        Checks if all GlobalIds are unique.
+
+        Only files containing unique GUIDs are valid for bim2sim.
+
+        Args:
+            id_list: list of all GUID's in IFC File
+        Raises:
+            TypeError: if loaded file does not have unique GUIDs
+            Warning: if uppercase GUIDs are equal
+        """
+        if len(id_list) > len(set(id_list)):
+            raise TypeError(
+                f"Loaded IFC file does not contain instances of type "
+                f"'IfcRelSpaceBoundary' but only files containing "
+                f"IfcRelSpaceBoundaries can be validated. Please ask "
+                f"the creator of the model to provide a valid IFC4 "
+                f"file.")
```

Comment:
The error message is copy paste from above. There should be something like no unique guids in ifc